content/docs/multi-user-overview.md

```diff
@@ -0,0 +1,144 @@
+deployments and other resources. The isolation mechanisms also prevent
+accidental deletion/modification of resources of other users in the deployment.
+Note that the isolation support in Kubeflow doesn't provide any hard security
+guarantees against malicious attempts by users to infiltrate other user's
+profiles.
+
+## Usage overview
+
+An administrator needs to deploy Kubeflow and configure the authentication.
+service for the deployment.  A user can log into the system and will by default
+be accessing their primary profile. Users have view and modify access to their
+primary profile. The owner of a profile can share access to the profile with
+another user in the system. When sharing the access to a profile, the owner can
+choose to share it as a view access only or view and modify access with the
+user.
+
+A user can select their active profile from the top bar on
+the Kubeflow Central Dashboard. Note that the user can only view the profiles
+in the dropdown list that they have view or modify access to.
+
+<img src="/docs/images/select-profile.png" 
+  alt="Select active profile "
+  class="mt-3 mb-3 border border-info rounded">
+
+For this overview we will illustrate the user isolation functionality using the
+Jupyter notebooks service which is the first service in the system to have full
+integration with the multi-user isolation functionality.  
+
+Once an active profile has been selected by the user, the Notebooks Manager UI
+will display only the active notebook servers in the currently selected
+profile. All other notebook servers remain hidden from the user. If they switch
+the active profile, the view will switch the list of active notebooks
+appropriately. The user can connect to any of the listed notebook servers and
+view and modify the existing Jupyter notebooks available in the server.
+
+For example, the following image shows the list of Notebook servers available
+in the user's primary profile:
+
+<img src="/docs/images/notebooks-in-profile.png" 
+  alt="List of notebooks in active profile "
+  class="mt-3 mb-3 border border-info rounded">
+
+And when an unauthorized user accesses the Notebooks in this profile, they are
+presented with an error.
+
+<img src="/docs/images/notebook-access-error.png" 
+  alt="Error listing notebooks in inacessible profile"
+  class="mt-3 mb-3 border border-info rounded">
+
+When the users create Jupyter notebook servers from the Notebooks Manager UI,
+the notebook pods are created in the active profile. If the user doesn't have
+modify access to the active profile, the user can only browse currently active
+notebook servers and access the existing jupyter notebooks but cannot create
+new notebook servers in that profile. Clearly the users can create notebook
+servers in their primary profile which they have view and modify access to.
+
+### Creating a user profile
+
+An **administrator** needs to create a profile for any user in the system.
```

Comment:
Please would you explain what you mean by an administrator.

Reply:
Updated the explanation of "administrator"